### PR TITLE
176364 setup stryker and codecoverage

### PR DESCRIPTION
**Pull request by nerellavinod**
Created the stryker job & published the report to pipeline and set code coverage threshold to 90% in the pipeline.

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -150,6 +195,14 @@ stages:
               PathtoPublish: "$(Build.ArtifactStagingDirectory)/codecoveragereport"
               ArtifactName: codecoveragereport
 
+          - task: BuildQualityChecks@9
+            displayName: "Code coverage fails if threshold exceeded"
```
Comment (harshal11869) [inline azure-pipelines.yml:199]:
```suggestion
            displayName: "Code coverage fails if threshold not meet"            
```

**Review by harshal11869 — CHANGES_REQUESTED** [review]

**Review by harshal11869 — APPROVED** [review]
LGTM.

File: azure-pipelines.yml (hunk 1 above)
Comment (nerellavinod) [reply azure-pipelines.yml]: resolved this suggestion.

**Review by harshal11869 — APPROVED** [review]

**Review by amogh14893 — APPROVED** [review]
LGTM

**Review by jrippington — APPROVED** [review]
Looks good, thanks!

**Pull request merged**